tests/test_dbutils.py

```diff
@@ -204,3 +204,23 @@ def test_secrets_get_and_redacting_logs(dbutils, mocker):
     inner.assert_called_with('GET', '/api/2.0/secrets/get', query={'key': 'bar', 'scope': 'foo'})
 
     assert value == 'hello'
+
+
+def test_jobs_task_values_set(dbutils):
+    dbutils.jobs.taskValues.set('key', 'value')
+
+
+def test_jobs_task_values_get(dbutils):
+    assert dbutils.jobs.taskValues.get('taskKey', 'key', debugValue='debug') == 'debug'
+
+    dbutils.jobs.taskValues.set('key', 'value')
+
+    assert dbutils.jobs.taskValues.get('taskKey', 'key', debugValue='debug') == 'debug'
```

Comment:
Can you add a comment here that we expect `get` to always return the `debugValue` when calling outside of a job context and not what was previously set using `set`?